### PR TITLE
chore: add trace messages

### PR DESCRIPTION
**Pull request by slavik-lvovsky**

### CODE REVIEW

File: packages/vscode-dependencies-validation/src/diagnostics.ts [hunk 1]
```diff
@@ -2,10 +2,13 @@ import type { DiagnosticCollection, Uri } from "vscode";
 import { Range, Diagnostic } from "vscode";
 import { isEmpty } from "lodash";
 import { findDependencyIssues } from "@sap-devx/npm-dependencies-validation";
+import { IChildLogger } from "@vscode-logging/types";
 import { getLogger } from "./logger/logger";
 import { NPM_DEPENDENCY_ISSUES_CODE } from "./constants";
 
-const logger = getLogger().getChildLogger({ label: "diagnostics" });
+export function logger(): IChildLogger {
```
Comment (bd82) [inline packages/vscode-dependencies-validation/src/diagnostics.ts:9]: why is this exported?

File: packages/vscode-dependencies-validation/src/logger/logger.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { ExtensionContext, OutputChannel } from "vscode";
+import type { ExtensionContext, OutputChannel } from "vscode";
```
Comment (bd82) [inline packages/vscode-dependencies-validation/src/logger/logger.ts:1]: 👍

File: packages/vscode-dependencies-validation/src/diagnostics.ts (hunk 1 above)
Comment (bd82) [reply packages/vscode-dependencies-validation/src/diagnostics.ts:9]: should it be exported via `internal` property if it is for tests purposes?

**Review by bd82 — APPROVED** [review]
see one minor in-lined comment

File: packages/vscode-dependencies-validation/src/diagnostics.ts (hunk 1 above)
Comment (slavik-lvovsky) [reply packages/vscode-dependencies-validation/src/diagnostics.ts:9]: Done

**Pull request merged**